morph.go

```diff
@@ -113,6 +114,25 @@ func (m *Morph) ApplyAll() error {
 	return nil
 }
 
+func sortMigrations(migrations []*models.Migration) []*models.Migration {
+	keys := make([]string, 0, len(migrations))
```

Comment:
can we not do something with `sort.Slice` like:

```go
	sort.Slice(pendingMigrations, func(i, j int) bool {
		return pendingMigrations[i].Name < pendingMigrations[j].Name
	})
```

Reply:
Right! Done in 0795872